content/html/concepts/tags/terms/dl/dl.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: '<dl>'
+Description: 'Dislpays terms and details, commonly for metadata purposes.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Lists'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<dl>` tag is a description list that displays terms and details, commonly for metadata purposes.
+
+## Syntax
+
+```html
+<dl>
+  <!-- <dt> terms here -->
+  <!-- <dd> details here -->
+</dl>
+```
+
+Inside the `<dl>` tag, one or more `<dt>` and `<dd>` tags can be used.
+
+**Note**: For Apple VoiceOver, the `<dl>` tag will be recognized and announced as a "list" on macOS devices whereas it [will not](https://cdpn.io/pen/debug/NzGaKP) be recognized on iOS devices.
+
+With iOS, the `<dl>` be recognized by may not always be the best option
+
+## Example
+
+The `<dl>` tag also supports nesting `<dt>`-`<dd>` pairs inside a [`<div>`](https://www.codecademy.com/resources/docs/html/tags/div) tag when working with content microdata, global attributes, or general [CSS](https://www.codecademy.com/resources/docs/css) styling.
+
+In the example below, this technique is used inside the `<dl>` tag. Additionally, some default styling is in place to highlight the list itself:
+
+```html
```

Comment:
I'm wondering if this example would benefit from a more real-world example of what terms and details might actually be used.

Reply:
I'm struggling to come up with one besides glossary/dictionary definitions. 😄